### PR TITLE
Add first unit test to Travis CI

### PR DESCRIPTION
**Pull request by carljparker**

### CODE REVIEW

File: easy_music_generator/preprocessor/tests/test_chord_distribution.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        my_chord_dict = {'chord_1': 1, 'chord_2': 1}
+
+        #
+        # Test the method as a static method
+        #
+        _ = cd.ChordDistribution.get_chord_probabilities(my_chord_dict)
+        self.assertEqual(my_chord_dict['chord_1'], 0.5)
+        self.assertEqual(my_chord_dict['chord_2'], 0.5)
+
+        #
```
Comment (andreiabds) [inline easy_music_generator/preprocessor/tests/test_chord_distribution.py:24]: I would remove this # before and after the comment

File: easy_music_generator/preprocessor/tests/test_chord_distribution.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        # Test the method as an instance method
+        #
```
Comment (andreiabds) [inline easy_music_generator/preprocessor/tests/test_chord_distribution.py:26]: remove

**Review by andreiabds — APPROVED** [review]
This is a great start! I would just remove the extra #. It looks cleaner when you only have # on the lines with comments.

**Comment by HasnahSaid** [comment]
I agree with Andreia about removing # before the comment. You can replace them with ''' comment ''' instead for longer comments which will look neater. But other than that, this looks good to me!

**Review by HasnahSaid — COMMENTED** [review]
I agree with Andreia about removing # before the comment. You can replace them with ''' ''' instead for longer comments which look neater. But other than that, this looks good to me!

**Pull request merged**